mjolnir_snap/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: mjolnir
+summary: Mjolnir snap
+description: A moderation tool for Matrix
+version: "1.6.4"
+base: core22
+license: Apache-2.0
+confinement: strict
+apps:
+  mjolnir:
+    command: 'bin/node $SNAP/mjolnir/index.js'
+    plugs:
+      - network
+parts:
+    node:
+        plugin: dump
+        source: https://nodejs.org/dist/v16.20.2/node-v16.20.2-linux-x64.tar.xz
+        stage:
+            - bin
+            - include
+            - lib
+    install-mjolnir:
+        after: [node]
+        plugin: nil
+        source: https://github.com/matrix-org/mjolnir.git
+        source-tag: v1.6.4
+        source-type: git
+        override-build: |
+            npm install -g yarn
+            yarn install
+            yarn build
```

Comment:
Can't the npm plugin be used to do this ?

Reply:
Since the snap command is "bin/node $SNAP/mjolnir/index.js", I preferred to install the node and use it in the build.